armi/materials/material.py

```diff
@@ -539,25 +539,44 @@ def removeLumpedFissionProducts(self) -> None:
     def getMassFracCopy(self):
         return copy.deepcopy(self.p.massFrac)
 
-    def checkTempRange(self, minV, maxV, val, label=""):
+    def checkPropertyTempRange(self, label, val):
+        r"""Checks if the given property / value combination fall between the min and max valid
+        temperatures provided in the propertyValidTemperature object.
+
+        Parameters
+        ----------
+        label : str
+            The name of the function or property that is being checked.
+
+        val : float
+            The value to check whether it is between minT and maxT.
+
+        Notes
+        -----
+        This was designed as a convience method for ``checkTempRange``.
+        """
+        (minT, maxT) = self.propertyValidTemperature[label][0]
+        self.checkTempRange(minT, maxT, val, label)
```

Comment:
@john-science Sorry for the super late comment on this PR...

I noticed that the standard syntax for entering into the `propertyValidTemperature` dict includes the units for the range.

However I also noticed that the above method doesn't do anything with those units.

Was it the intention to do some internal conversion on the units here to match whatever was passed in?